src/map/script.cpp

```diff
@@ -15410,38 +15410,43 @@ BUILDIN_FUNC(isday)
 BUILDIN_FUNC(isequippedcnt)
 {
 	TBL_PC *sd;
-	int i, id = 1;
-	int ret = 0;
 
-	if (!script_rid2sd(sd)) { //If the player is not attached it is a script error anyway... but better prevent the map server from crashing...
+	if (!script_rid2sd(sd)) {
 		script_pushint(st,0);
 		return SCRIPT_CMD_SUCCESS;
 	}
 
-	for (i=0; id!=0; i++) {
-		short j;
-		FETCH (i+2, id) else id = 0;
-		if (id <= 0)
+	int ret = 0;
+	int total = script_lastdata(st);
+	std::vector<int32> list;
+
+	for ( int i = 2; i <= total; ++i ) {
```

Comment:
```suggestion
	for (int i = 2; i <= total; ++i) {
```
If you are not going to put a leading space and a trailing space in every parenthesis in this function, don't put one in the for loop.